### ISSUE TITLE
App doesnt build for device when linphone target's deployment target is 9.0

### ISSUE THREAD

**Issue by nceruchalu** [BelledonneCommunications/linphone-iphone#229]
Hi there,

Trying to set the linphone target's deployment target to 9.0 fails to build for a **device**.
I get the following error message:
```
ld: warning: object file (/path/linphone-iphone/liblinphone-sdk/apple-darwin/lib/libopus.a(celt_pitch_xcorr_arm-gnu.S.o)) was built for newer iOS version (10.2) than being linked (9.0)
ld: warning: object file (/path/linphone-iphone/liblinphone-sdk/apple-darwin/lib/libopus.a(armopts-gnu.S.o)) was built for newer iOS version (10.2) than being linked (9.0)
ld: '/path/linphone-iphone/liblinphone-sdk/apple-darwin/lib/libopus.a(celt_pitch_xcorr_arm-gnu.S.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
Would you have any suggestions for how to get around this?

Thanks,
Noddy

**Comment by nceruchalu** [comment]
Hello @ReisBenjamin is there any way you could comment on this?

**Comment by ReisBenjamin** [comment]
Hello,
This issue has nothong to do with de deployment target. You're trying to link with code a file compiled without bitcode.
You can for now disable bitcode while we try to fix this issue
Regards

**Comment by nceruchalu** [comment]
Thanks for the explanation!
I think I'll wait while you try to fix the issue 😄

**Comment by ReisBenjamin** [comment]
fixed in commit 68607efa15f5e6a05b4e6ef096c96525fab4ea94
You'll need to pull and update your submodules then remake the SDK

Best regards
